.travis.yml

```diff
@@ -0,0 +1,47 @@
+osx_image: xcode7.3
+sudo: required
+dist: trusty
+language: c
+matrix:
+  include:
+  - os: linux
+    env: CC=clang CXX=clang++ npm_config_clang=1
+    compiler: clang
+  - os: osx
+cache:
+  directories:
+  - node_modules
+  - app/node_modules
+  - "$HOME/.electron"
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    - llvm-toolchain-precise-3.7
+    packages:
+    - libgnome-keyring-dev
+    - icnsutils
+    - clang-3.7
+    - graphicsmagick
+    - xz-utils
+    - rpm
+before_install:
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update ; fi
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install git-lfs; fi
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then git lfs pull; fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then curl -L https://github.com/github/git-lfs/releases/download/v1.2.0/git-lfs-linux-amd64-1.2.0.tar.gz | tar -xz; fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then git-lfs-1.2.0/git-lfs pull; fi
+install:
+- nvm install 6
+- npm install npm -g
+- npm install electron-builder@next
+- npm install
+- npm prune
+script:
+- npm run dist
+branches:
+  except:
+  - "/^v\\d+\\.\\d+\\.\\d+$/"
+env:
+  global:
+    secure: MaZ/wVvbLeu2/Qjwv+eItPq0ThMLEGqXqOET+XpIrFFzIbB5Wy9zMsIPqXbJb4k17xr9YwkCpgU8/myi5wR9g+pjDcHG7Q1fQ1PFMQPSRamAWreP0He0Xs9SVnELZ1P7hEgSMcum2YT2dPtSsYJ2xS8GsBNTNSE5Hrdb0rjXfcg6cSHRqpq4HRYppUFQ6z+oMRlh6X0ThaDwuIOhON+ZTpvhGlgPpuKJSXCH2khn0KYGGUM9C3tPdhh3W3T04ujEOxCWGLpxuFzm/C14uywzha9dTdm+iYnj8JeFYonaeW1sHI0V99BfIZC+bLWTBcaUiPL6Fk6CUzg1jQ4HjBVZBXGryA3pkwwTT3Zps7ORXAKd0MeHe/XWx05gWgM0zQhjy5nyd7K3dQCcEOXFhy1CCEC8OdPIoNGnOBmGYxr0nxtc8SVZS5I1A1rfp/kerUbKRLBinXEb7kewuUhuPyuA6eYEDl41QSbU0c32tl+fQng3ffyVIGXs6DEd+Zl0oELO8HCgG5IJJVa+iB3ixgP11YYsz/dXIYO8NT4dFEbmP/w1hpReysTkCOSYQ6aZwOiPMpR7dpnmliICEE5M91iC0Dz2MXjz7So3qxfbDGX2WOmKPfGoFPhrfiNOI5+5OnaCheBb93PdIVDzPoB7PR1NgH7ZX0zO2uqsO5+3kWuOszw=
```

Comment:
It is the encrypted environment variable GH_TOKEN which allows Travis to publish github releases, when you enable Travis builds for this repo we should probably use the system on the site and not in Travis.yml